src/app/view/perspectives/table/body/rows/cell-group/table-cell-group.component.ts

```diff
@@ -147,4 +147,8 @@ export class TableCellGroupComponent implements OnChanges {
     this.store$.dispatch(new TablesAction.SetCursor({cursor}));
     event.stopPropagation();
   }
+
+  public onAffect() {
+    // just to run change detection
```

Comment:
I am just curious: Output events run change detection?

Reply:
Yes, of course. We need to run it here because a child component uses `HostBinding` decorator and when its property is changed, the view is not refreshed if you run the change detection only in the child component itself since those properties behave the same as they would if you set them in the parent template.